scripts/uploadAgreements.js

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable no-console */
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+const ipfsAPI = require("ipfs-api");
+const fs = require("fs");
+const path = require("path");
+
+const isFilePinned = async (ipfs, hash) => {
+  const matchedHashes = (await ipfs.pin.ls()).filter(response => {
+    if (response.hash === hash) return true;
+    return false;
+  });
+  return matchedHashes.length > 0;
+};
+
+const addFiletoIpfs = async (ipfs, file, name) => {
+  try {
```

Comment:
do we need try...catch here? just let exception go. you will catch it later

Reply:
done